eservice/lib/libpdo_enclave/CMakeLists.txt

```diff
@@ -126,7 +125,6 @@ SET(CMAKE_SHARED_LIBRARY_PREFIX "")
 ADD_LIBRARY(${PROJECT_NAME} SHARED  ${PROJECT_HEADERS} ${PROJECT_SOURCES} ${PROJECT_EDGE_SOURCES} ${PROJECT_EDL})
 SGX_SIGN_ENCLAVE(${PROJECT_NAME} ${PDO_ENCLAVE_CODE_SIGN_PEM} ${PROJECT_CONFIG})
 
-TARGET_LINK_LIBRARIES(${PROJECT_NAME} "${PDO_TOP_DIR}/common/build/libtpdo-common.a")
 TARGET_LINK_LIBRARIES(${PROJECT_NAME} "${PDO_TOP_DIR}/common/build/libpdo_contracts.a")
```

Comment:
this has to be there in order to regenerate the library if something in common changes

Reply:
That's a duplicate line -- `tpdo-common` is also linked at line 139.
You probably mean that we are missing an `add_dependencies`. I will include that.

Reply:
actually they aren't the same. if you just include the library in the list then a change to common will not re-link the library in the enclave. you need to have the FILE included. as you have it, a change to common will NOT cause the enclave to be relinked.

Reply:
you're right, and me too!
I have restored that line, and removed the duplicate `tpdo-common` link that appeared below.